site/app/controllers/NavigationController.php

```diff
@@ -26,7 +27,13 @@ public function noAccess() {
      * @Route("/courses/{_semester}/{_course}", requirements={"_semester": "^(?!api)[^\/]+", "_course": "[^\/]+"})
      */
     public function navigationPage() {
-        $gradeables_list = new GradeableList($this->core);
+        try {
+            $gradeables_list = new GradeableList($this->core);
+        }
+        catch (DatabaseException $e) {
```

Comment:
Please add a call of `ExceptionHandler::handleException($e);` here so that sysadmins have a meaningful log output to act upon.

Reply:
Changed.